database/seeds/FAQSeeder.php

```diff
@@ -14,7 +14,7 @@ public function run() {
 
 		$question_answers_en = [
 			'I want to offer a job to a student, how to proceed?'	=>
-				'You simply have to create an ad following <a href="nouveau-job">
+				'You simply have to create an ad following <a href="new-job">
```

Comment:
Using `' + action('AdController@created') + ' ` could provide a better solution for long-term, same below.